### PR TITLE
release: v0.1.6

### PR DESCRIPTION
**Pull request by tmpbeing**

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,6 +1,12 @@
 Changelog
 ---------
 
+0.1.6 (2024-04-25)
+******************
+
+- :py:meth:`Optimization.run()<ansys.simai.core.data.optimizations.OptimizationDirectory.run>` will now error out if no workspace is provided and no configured one could be found
```
Comment (yias) [inline CHANGELOG.rst:7]: suggestion:  ... now raises an exception if no workspace is provided or no configured workspace is found

File: CHANGELOG.rst [hunk 2]
```diff
@@ -1,6 +1,12 @@
...
+- Fix potential RecursionError on authentication refresh
```
Comment (yias) [inline CHANGELOG.rst:8]: suggestion: Fixed RecursionError after authentication refresh.

**Review by yias — COMMENTED** [review]
just a couple suggestions from my side. I think @romane-m input would be valuable here

**Review by yias — APPROVED** [review]
It looks fine, but I think having an opinion from documentation would be nice.

File: CHANGELOG.rst (hunk 1 above)
Comment (romane-m) [reply CHANGELOG.rst:7]: Hi @tmpbeing, two suggestions (but feel free to ignore if we loose information by being too synthetic):

- The method :py:meth:`Optimization.run()<ansys.simai.core.data.optimizations.OptimizationDirectory.run>` now raises an exception if no configured workspace is provided or found. 

- The method :py:meth:`Optimization.run()<ansys.simai.core.data.optimizations.OptimizationDirectory.run>` now raises an exception if no workspace is provided or none is configured.

**Review by kliment-slice — APPROVED** [review]

File: CHANGELOG.rst [hunk 3]
```diff
@@ -1,6 +1,19 @@
...
+******************
+
+New:
+
+- Added new method :py:meth:`TrainingData.assign_subset()<ansys.simai.core.data.training_data.TrainingData.assign_subset>` that allows you to assign a Train, Validation, or Test subset to your data.
+
+
+Fixes:
+
+- The method :py:meth:Optimization.run()<ansys.simai.core.data.optimizations.OptimizationDirectory.run> now raises an exception if no workspace is provided and none is configured.
```
Comment (kliment-slice) [inline CHANGELOG.rst:14]: quotes for method after meth:`

**Pull request merged**